GOVERNANCE.md

```diff
@@ -0,0 +1,45 @@
+# Kroxylicious Governance
+
+This document outlines the governance model for Kroxylicious.
+This governance model is designed to uphold the principles of transparency, open collaboration, and community involvement.
+
+[cfc]: https://www.commonhaus.org/bylaws/cf-council.html
+[cdm]: https://www.commonhaus.org/bylaws/decision-making.html
+[ctp]: https://www.commonhaus.org/policies/trademark-policy/
+[coc]: ./CODE_OF_CONDUCT.md
+[contrib]: ./CONTRIBUTING.md
+
+## Table of Contents
+
+- [Roles and Responsibilities](#roles-and-responsibilities)
+- [Decision Making](#decision-making)
+- [Code of Conduct](#code-of-conduct)
+- [Trademark Policy](#trademark-policy)
+- [Contributing](#contributing)
+
+## Roles and Responsibilities
+
+- **Code owners:** Individuals with write access to the repository, responsible for reviewing and merging contributions, and steering project direction. These individuals (or groups) will be maintained in the repository (`CODEOWNERS` or `.github/CODEOWNERS`).
```

Comment:
I think we should adopt the CODEOWNERS convention (rather than relying on the pom.xml)

Reply:
I think thats a good call really. I'd kinda like to sync the two as the POM is published with the jars to maven central so kinda feels like we should keep them in sync.

Reply:
Are you suggesting automation or just a comment in the files?

Reply:
I was thinking of the code owners file as authoritative and then using that to inject the developers node into the Pom but that's probably overkill. We just need to update both when adding a new code owner.

So I think my position is we maintain both, but happy for docs etc to point at the file as opposed to the node.